lib/pbench/server/cache_manager.py

```diff
@@ -610,58 +515,25 @@ class CacheManager:
 
             /srv/pbench/archive/fs-version-001/
 
-        This directory is linked from /var/www/html/pbench-tarballs for Apache
-        access.
-
         A directory is created under this root for each controller name used by
         a dataset. Within each ARCHIVE controller directory you'll find:
 
             A set of Pbench Agent dataset results, each comprising a ".tar.xz"
             tar archive (conventionally referred to as a "tarball") and a
             ".tar.xz.md5" MD5 file with the same base name.
 
-    INCOMING
-
-        The INCOMING tree is rooted under the Pbench "top dir" public_html
-        directory, conventionally something like
-
-            /srv/pbench/public_html/incoming/
-
-        It's also linked from /var/www/html/incoming.
-
-        Like the ARCHIVE tree, it will contain a directory for each controller;
-        but a controller has a subdirectory for each dataset result, with the
-        root name of the tarball (without the trailing `.tar.xz`), that
-        contains the unpacked contents of the tarball.
-
-        This exists only after the pbench-unpack-tarballs script has run (based
-        on the TO-UNPACK state link).
-
-    RESULTS
-
-        The RESULTS tree is rooted under the Pbench "top dir" public_html
-        directory, conventionally something like
+    CACHE
 
-            /srv/pbench/public_html/results/
+        The CACHE tree is rooted under the Pbench "top dir" directory,
+        conventionally something like
 
-        It's also linked from /var/www/html/results.
+            /srv/pbench/cache/
```

Comment:
Similar references elsewhere have a dot at the beginning of `cache`...should there be one here?

Reply:
Referencing config settings in code is always at risk of getting stale. Yeah, I wrote this before I decided to hide the cache root by default. As an abstract example, there's nothing wrong with this, but matching the default value wouldn't be an entirely bad thing...